### DISCUSSION TITLE
Adjusting size of images

### DISCUSSION THREAD

**Discussion by LenaRampula** [h2oai/wave#721] [Q&A]
How can the image size in an image card be controlled?
Changing the size of the original image doesn't seem to have any effect.

**Comment by lo5** [comment]
@LenaRampula - Looks like the card always tries to resize the image to the card size. 
@mturoci - Any idea why so? This commit (https://github.com/h2oai/wave/commit/3d526a2ce8b5b57a2d33af4c87f214e83e7cda4e) introduced `object-fit`, but the issue has no details about the change in behavior: https://github.com/h2oai/wave/issues/330.

**Reply by mturoci** [reply]
I considered this expected behavior - fill all the card's available space if possible, otherwise the card would look ugly with all the whitespace (images smaller than card) or scrollbars (images larger than card). This is default behavior for flex layouts as well. The only way to adjust the size today is to change the card's size. We can also use intrinsic size if considered better, but it is not IMO. Another alternative is to expose `img_width` and `img_height` props.

@lo5 wdyt?

**Reply by LenaRampula** [reply]
The image didn't adjust to the size of the card. It was always much smaller, even if the actual dimensions of the image changed. Fixed it by uploading the image and using markdown card.
